react/Button/Button.less

```diff
@@ -75,10 +97,45 @@
   .buttonColor(@sk-black, @sk-mid-gray-light);
 }
 
+.root_isGhost,
 .root_isTransparent {
-  color: @sk-link;
   background-color: transparent;
   box-shadow: none;
+}
+
+.root_isGhost {
+  border: 2px solid;
+  font-weight: @sk-bold;
+  line-height: 0;
```

Comment:
I'm wondering if setting line-height to `normal` might be a better option here? From reading through the code this value didn't feel right to me. I guess another option could be to set it to 45px - 4px of border?